VERSIONING.md

```diff
@@ -0,0 +1,90 @@
+# Versioning
+
+This document describes the versioning policy for this repository. This policy
+is designed so that the following goal can be achieved:
+
+**Users are provided a codebase of value that is stable and secure.**
+
+## Policy
+
+* Versioning of this project will be idiomatic of a Go project using [Go
+  modules](https://github.com/golang/go/wiki/Modules).
```

Comment:
Maybe, we can link to https://golang.org/ref/mod#versions instead of the wiki. Modules wiki is more of a tracker, tutorial and an FAQ. The versioning is more formally defined on the reference page.